Apps/WebClient/src/Server/Models/BannerChangeEvent.cs

```diff
@@ -0,0 +1,41 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Models
+{
+    using System;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// Object that represents a banner change push.
+    /// </summary>
+    public class BannerChangeEvent
```

Comment:
This object looks general enough. Thoughts on making a base change event and subclassing for the data field?

Reply:
Yes, I was working on it as a base class but due to some complexities I removed it for the time being.  When/if we do this again we can then abstract it out.